### ISSUE TITLE
Disabled NcInput has insufficient text contrast for accessibility

### ISSUE THREAD

**Issue by GVodyanov** [nextcloud/nextcloud-vue#8141]
This contrast here with the background isn't sufficient:

<img width="827" height="99" alt="Image" src="https://github.com/user-attachments/assets/8a6b5853-603d-44c0-a169-aab91136fae2" />

**Comment by ShGKme** [comment]
They don't have to be contrast since they are disabled. We have the same for many interactive elements, and this even was certified in this design. 

Ref: https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum

> Inactive User Interface Components
>
> User Interface Components that are not available for user interaction (e.g., a disabled control in HTML) are not required to meet contrast requirements. An inactive user interface component is visible but not currently operable. An example would be a submit button at the bottom of a form that is visible but cannot be activated until all the required fields in the form are completed.
